sql/core/src/test/resources/sql-tests/results/show-create-table.sql.out

```diff
@@ -245,3 +245,103 @@ DROP TABLE tbl
 struct<>
 -- !query 26 output
 
+
+
+-- !query 27
+CREATE TABLE tbl (a INT, b STRING, c INT) USING parquet
+-- !query 27 schema
+struct<>
+-- !query 27 output
+
+
+
+-- !query 28
+CREATE VIEW view_SPARK_30302 (aaa, bbb)
+AS SELECT a, b FROM tbl
+-- !query 28 schema
+struct<>
+-- !query 28 output
+
+
+
+-- !query 29
+SHOW CREATE TABLE view_SPARK_30302
+-- !query 29 schema
+struct<createtab_stmt:string>
+-- !query 29 output
+CREATE VIEW `view_SPARK_30302`(`aaa`, `bbb`)
+AS SELECT a, b FROM tbl
+
+
+-- !query 30
+DROP VIEW view_SPARK_30302
+-- !query 30 schema
+struct<>
+-- !query 30 output
+
+
+
+-- !query 31
+CREATE VIEW view_SPARK_30302 (aaa COMMENT 'comment for aaa', bbb)
+COMMENT 'This is a comment for view'
+AS SELECT a, b FROM tbl
+-- !query 31 schema
+struct<>
+-- !query 31 output
+
+
+
+-- !query 32
+SHOW CREATE TABLE view_SPARK_30302
+-- !query 32 schema
+struct<createtab_stmt:string>
+-- !query 32 output
+CREATE VIEW `view_SPARK_30302`(`aaa` COMMENT 'comment for aaa', `bbb`)
```

Comment:
In hive, the format is like this;
```
hive> show create table t;
OK
CREATE TABLE `t`(
  `a` int COMMENT 'comment for a', 
  `b` int)
COMMENT 'a\'aaaaa'
...
```
A single column info per line looks more readable to me. WDYT?

Reply:
Yes I also think that's better, but should we change it in a separate pr? In this pr, let's keep the same format as current hive table and data source table. How about that?

Reply:
Yea, that looks ok.